packages/env/lib/env.js

```diff
@@ -411,13 +411,17 @@ async function configureWordPress( environment, config ) {
 	// Install WordPress.
 	await dockerCompose.run(
 		environment === 'development' ? 'cli' : 'tests-cli',
-		`wp core install
-			--url=localhost:${ port }
-			--title='${ config.name }'
-			--admin_user=admin
-			--admin_password=password
-			--admin_email=wordpress@example.com
-			--skip-email`,
+		[
+			'wp',
+			'core',
+			'install',
+			`--url=localhost:${ port }`,
```

Comment:
Unrelated: I have a use-case I'm wondering if there is some better support for, or if we could make better support for. Specifically, to test Internet Explorer, I use [VirtualBox](https://www.virtualbox.org/) and a [Microsoft-provided VM](https://developer.microsoft.com/en-us/microsoft-edge/tools/vms/). To test the Gutenberg site from the host machine, I find that I have to go to Settings > General and change the `url` from `localhost` to the IP address of my host machine on my local network (`192.168.1.162`). If we have configurations for other values (like port), is it something where we could allow configuration of the hostname here as well? Would that work as well as I'm expecting it to, if I would have a `.wp-env.override.json` in my local Gutenberg directory?

Reply:
Hm, interesting. Yes we could add `siteURL` to `.wp-env.json` and then you would set it in `.wp-env.override.json` for this use case. I'm not sure if it's worth adding this functionality though given that the URL can be changed easily via WP Admin.

@tellthemachines: I remember you running into this. What did you end up doing? Do you map `localhost` in VirtualBox to the host machine's IP?